CMS/cms-basics

```diff
@@ -0,0 +1,345 @@
+---
+title: CMS basics
+---
+
+## Introduction
+
+Skygear CMS aims to provide a user friendly-interface for business user to update the content of the app. As a developer, you can configure what to display / be editable in the CMS with the YML file at the Developer Portal.
+
+Some of you may wonder what's the difference between the CMS and the data browser, here's a summary.
+
+
+|Difference	|Database Browser	|CMS	|
+|---	|---	|---	|
+|Target user	|Developers who need to view and alter database data and schema.	|System Admin or managers who need to view and update content of the system.	|
+|Data views	|Showing all tables, including Skygear default tables such as _auth, _role ,etc. Also shows every fields including default fields in each table, such as _access, _id etc.	|Default hide all tables. Developers need to configure which table and fields to be shown at	|
+|Data export	|Yes	|Yes	|
+|Data import |No |Yes |
+|Who can access	|Collaborators of your app on Skygear	|Admin users in Skygear user role record	|
+|Customizable	|No. It default shows all tables and columns.	|Yes, with config in YML format.	|
+
+
+## Enable CMS plugin at Skygear Portal
+
+By default, CMS is not enabled on the portal. You can turn on the the CMS plugin at `Skygear Portal > CMS. `If you are not a new Skygear users and wanted to use this new version of CMS, re-enable the CMS plug-in 🤓
+
+## CMS configuration overview
+
+Skygear supports a simple way to configure CMS via editing the CMS Configuration YML. You can find the current config YML at ` Skygear Portal > CMS > CMS Configuration`.
+
+The YML itself looks explanatory, still there are some details we want you to know. Below is a simple overview of what you can configure in the CMS.
+
+```
+site:
```

Comment:
The example should be a valid yaml. So I suggest we change it to something like these:

```
site: # pages in the CMS, each page will be shown on the left sidebar.
  - type: Record
    name: user
    label: User

records:
  user: # Configure the user record
    list: # Displayable fields on the first page
[... etc]
```